src/components/capture/Create.tsx

```diff
@@ -121,15 +119,17 @@ function CaptureCreate() {
         EditorStoreState<DraftSpecQuery>['id']
     >((state) => state.id);
 
+    // Reset the cataog if the connector changes
```

Comment:
> Reset the cataog if the connector changes

There is a typo in the word _catalog_ in this comment. This does not need to be changed. Calling this out for general awareness.